tests/mtd_at25xxx/main.c

```diff
@@ -73,8 +73,8 @@ static void test_mtd_write_erase(void)
     ret = mtd_erase(dev, TEST_ADDRESS, dev->pages_per_sector * dev->page_size);
     TEST_ASSERT_EQUAL_INT(0, ret);
 
-    uint8_t expected[sizeof(buf_read)];
-
+    uint8_t expected[sizeof(buf_read)] = {0};
+    memset(expected, 0 , sizeof(expected));
```

Comment:
No need to initialize the buffer with 0 twice if it's being to right afterwards.